react/tic-tac-toe/src/app/screens/Game/index.js

```diff
@@ -55,6 +56,9 @@ class Game extends Component {
         <div className={styles.gameinfo}>
           <ol>{moves}</ol>
         </div>
+        <div className={styles.gameInfo}>
```

Comment:
Think the interface as a group of visual items. This div should be part of MatchesList